packages/x-tree-view/src/models/items.ts

```diff
@@ -4,3 +4,9 @@ export type TreeViewItemId = string;
 export type TreeViewBaseItem<R extends {} = { id: string; label: string }> = R & {
   children?: TreeViewBaseItem<R>[];
 };
+
+export type TreeViewItemsReorderingAction =
```

Comment:
This type need to be on the community package because it is used in `TreeItem2DragAndDropOverlay` which is on the community package (to avoid creating a different Tree Item component on the pro package)